src/components/ProfileApp/network.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ENDPOINT_USER_GET_PROFILE,
+  ENDPOINT_USER_UPDATE_PROFILE
+} from '../../config/api';
+import { TYPE_GET, TYPE_POST } from '../../config/global';
+import { fetchBackend } from '../../utils/network';
+
+// Fetches user profile from the server.
+
+export const fetchGetUserProfile = async () => {
```

Comment:
this function should take one arg: `username`. `get`ting a user profile may be done by anyone (profile page can be a public page).